packages/renderer/src/lib/ContainerListCompose.spec.ts

```diff
@@ -0,0 +1,136 @@
+/**********************************************************************
```

Comment:
it seems it's missing the case when we have an error when starting a compose group

Reply:
Will add this in PR https://github.com/containers/podman-desktop/issues/3399

I made tests with https://github.com/cdrage/podman-desktop/tree/error-tests and `searchByText` but it does not work as error callbacks aren't implemented correctly.